### PR TITLE
use version property used in KIE parent

### PR DESCRIPTION
**Pull request by mareknovotny**
See https://github.com/kiegroup/jbpm-work-items/security/dependabot/pom.xml/com.google.guava:guava/open
Trying to unify on the same version we have in KIE parent

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by mareknovotny** [comment]
jenkins retest this please

**Review by gmunozfe — APPROVED** [review]
Looks good to me

File: pom.xml [hunk 1]
```diff
@@ -65,7 +65,7 @@
     <version.org.xhtmlrenderer>9.1.15</version.org.xhtmlrenderer>
     <version.bouncycastle>1.67</version.bouncycastle>
     <version.xmlsec>1.5.1</version.xmlsec>
-    <version.google.guava>29.0-jre</version.google.guava>
+    <version.google.guava>${version.com.google.guava}</version.google.guava>
```
Comment (afalhambra) [inline pom.xml:68]: why not just remove the guava dep from the dependencyManagement section of the pom.xml and leave it the one managed at KIE parent? so we can ensure we're always aligned with the right version, wdyt?

Comment (mareknovotny) [reply pom.xml:68]: we could, even this doesn't change too much and we have still option to overwrite it here for any reason we need.

**Pull request merged**